Packs/SplunkPy/Scripts/SplunkPySearch/SplunkPySearch.yml

```diff
@@ -0,0 +1,29 @@
+commonfields:
```

Comment:
Please generate a readme file for this script.
You can use the command demisto-sdk generate-docs -i <yml file path>